java/lang/os_command_injection.yml

```
@@ -0,0 +1,65 @@
+imports:
+  - java_shared_lang_instance
+patterns:
+  - pattern: |
+      $<RUNTIME_INIT>.exec($<DYNAMIC_INPUT>);
+    filters:
+      - variable: RUNTIME_INIT
+        detection: java_lang_os_command_injection_runtime
+      - variable: DYNAMIC_INPUT
+        detection: java_lang_os_command_injection_dynamic_input
```

This will avoid ternary conditions matching. eg. `x.exec(dyn ? "const" : "const2")`

```suggestion
        detection: java_lang_os_command_injection_dynamic_input
        scope: result
```